assets/section-blog-post.css

```
@@ -175,3 +175,7 @@
     margin: 5rem 0 8rem;
   }
 }
+
+.article-template__back:last-child {
+  margin-bottom: 2.5rem;
```

```suggestion
  margin-bottom: 3.2rem;
```
Keeping it in our multiple of 4 and safe space to be accompanied with other default section setting values from other sections.